cluster/images/hyperkube/static-pods/etcd.json

```diff
@@ -10,7 +10,7 @@
     "containers": [
       {
         "name": "etcd",
-        "image": "gcr.io/google_containers/etcd-ARCH:2.3.7",
+        "image": "gcr.io/google_containers/etcd-ARCH:2.2.1",
```

Comment:
OK. If that's the only user of this file, and it started out as 2.2.5, then it should be fine to change it back to 2.2.5. Sorry, it was lost in the three PR dance we did.